test/e2e/basic_scheduling/basic_scheduling_test.go

```diff
@@ -70,6 +70,16 @@ var _ = ginkgo.Describe("", func() {
 		Ω(err3).NotTo(HaveOccurred())
 		Ω(d).NotTo(BeNil())
 
+		ginkgo.By("Restart scheduler pod")
+		_, err4 := kClient.ScaleDeployment(configmanager.YKScheduler, 0, configmanager.YuniKornTestConfig.YkNamespace)
+		gomega.Ω(err4).NotTo(gomega.HaveOccurred())
+		err5 := kClient.WaitForPodBySelectorTerminated(configmanager.YuniKornTestConfig.YkNamespace, fmt.Sprintf("component=%s", configmanager.YKScheduler), 60)
```

Comment:
I'm really not sure why this timeout had to be 60. Waiting for pods to be fully terminated (this does not correspond to "terminating" in k8s state diagram, but rather failing an existence check for any pods with the `selector` labels in the given `namespace`) takes around 10-12 seconds. But even setting 30 here leads to a `timeout exceeded` error. Wasn't sure how to debug this or estimate the value more properly.